react/typings/vtex.document-field.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'vtex.document-field'
```

Comment:
why you need this?

Reply:
Typescript message: "Cannot find module 'vtex.document-field'."

Reply:
did you run `vtex setup` after adding it to your dependencies on `manifest.json`?